packages/cascara/src/lib/mock/fakeData.js

```diff
@@ -15,11 +15,6 @@ export const generateFakeEmployees = (qty) =>
       country: faker.random.arrayElement(['USA', 'Brazil', 'Argentina']),
       eid: faker.datatype.uuid(),
       email: faker.internet.email(firstName, lastName, 'espressive.com'),
-      employeeNumber: faker.datatype.number(),
-      fullName: `${firstName} ${lastName}`,
-      homePhone: faker.phone.phoneNumber(),
-      officePhone: faker.phone.phoneNumber(),
-      title: faker.name.jobTitle(),
```

Comment:
@lordpixel this looks like a good optimization and something to think about for future tests.

This has me wondering... how long would the tests take to run on average if this data was just JSON and not being generated from a JS library? I think we should consider that. If static data is more performant, we might want to say we limit our actual tests to static data only and only use generated data in development. I don't know the answer to this either way.

Reply:
I didn't think about removing `faker` from the equation, but then again, why do we use these kind of tools if at the end we are going to remove them just because tests take 3 seconds more to run?

Shouldn't we think twice before adding libraries, fixtures and all these things?

Reply:
Faker is a helpful tool if you do not yet know what data to use. Or if you have a use case where you need a LOT of data. It is just a tool like anything else. Part of our job is figuring out which tool to use in each use case. I think Faker still has use cases for Cascara repo, but it might not be good specifically for tests if we find out it is not as performant as we want.